docs/mllib-guide.md

```diff
@@ -121,6 +121,9 @@ Deprecations:
  We encourage users to use `spark.ml.regression.LinearRegresson` and `spark.ml.classification.LogisticRegresson`.
 * [SPARK-14900](https://issues.apache.org/jira/browse/SPARK-14900):
  In `spark.mllib.evaluation.MulticlassMetrics`, the parameters `precision`, `recall` and `fMeasure` have been deprecated in favor of `accuracy`.
+* [SPARK-15644](https://issues.apache.org/jira/browse/SPARK-15644):
+ In `spark.ml.util.BaseReadWrite`, the `context` method has been deprecated in favor of `session`.
```

Comment:
Could you please list this as MLReader and MLWriter instead of BaseReadWrite?  Those are the public APIs.